dandiapi/zarr/tests/test_ingest_zarr_archive.py

```diff
@@ -88,7 +88,7 @@ def test_ingest_zarr_archive_assets(zarr_archive_factory, zarr_file_factory, dra
     # Assert asset size, metadata
     assert asset.size == 0
     assert asset.full_metadata['contentSize'] == 0
-    assert asset.full_metadata['digest']['dandi:dandi-zarr-checksum'] is None
+    assert asset.full_metadata['digest']['dandi:dandi-zarr-checksum'] == ''
```

Comment:
Note, this change does leak out via the HTTP API, as shown. I'm not sure if that's a compatibility problem with anything.

Reply:
This will most likely require a corresponding change in the dandi CLI